integration_test/features/steps/fetch_step.py

```diff
@@ -0,0 +1,72 @@
+import os
+import sys
+import time
+
+from hamcrest import assert_that, is_, is_not
+
+
+PARENT_PATH = os.path.abspath("..")
+if PARENT_PATH not in sys.path:
+    sys.path.insert(0, PARENT_PATH)
+
+
+@when('I fetch "{profile}" SVID')
+def step_impl(context, profile):
+    c_client_bin = os.popen("../build/workload/c_client svid_type=%s" % profile.lower())
+    context.result = c_client_bin.read()
+
```

Comment:
I liked this approach! reusing the profile for fetching both x509 and JWT. 
Well done!